exchanges/stream/websocket.go

```diff
@@ -376,63 +373,57 @@ func (w *Websocket) dataMonitor() {
 }
 
 // connectionMonitor ensures that the WS keeps connecting
-func (w *Websocket) connectionMonitor() error {
-	if w.checkAndSetMonitorRunning() {
```

Comment:
`checkAndSetMonitorRunning` method and `errAlreadyRunning` can be removed completely.

Reply:
removed